mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMessageIdManager.java

```diff
@@ -34,12 +34,14 @@
 import javax.mail.Flags;
```

Comment:
Detailed test of this changeset will be provided after rebasing https://github.com/linagora/james-project/pull/1042